boardswarm-cli/src/ui_term.rs

```diff
@@ -56,6 +74,20 @@ impl Widget for UiTerm<'_> {
             let y = area.y;
             buf.set_span(x, y, &span, width);
         }
+
+        if self.state_shown {
+            let str = format!("{:?}", self.terminal_state);
+            let width = str.len() as u16;
+            let style = Style::reset().fg(Color::Black);
+            let style = match self.terminal_state {
+                TerminalState::PowerOff => style.bg(Color::LightRed),
+                TerminalState::PowerOn => style.bg(Color::LightGreen),
+                TerminalState::PowerReset => style.bg(Color::LightYellow),
+                TerminalState::Initial => style.bg(Color::Gray),
+            };
+            let span = Span::styled(str, style);
+            buf.set_span(area.width - width, area.height - 1, &span, width);
```

Comment:
this probably should not be in the uiterm widget but instead in e.g. a status bar widget